synapse/handlers/auth.py

```diff
@@ -263,7 +293,7 @@ def _get_session_info(self, session_id):
         if not session_id:
             # create a new session
             while session_id is None or session_id in self.sessions:
-                session_id = stringutils.random_string(24)
+                session_id = stringutils.random_string_with_symbols(24)
```

Comment:
Yeah, maybe - this one only goes into json so it should be fine, but possibly the extra token space isn't worth it.